apps/web/src/components/forms/status-page-form.tsx

```diff
@@ -175,6 +195,44 @@ export function StatusPageForm({
             </FormItem>
           )}
         />
+        <FormField
+          control={form.control}
+          name="icon"
+          render={({ field }) => (
+            <FormItem className="sm:col-span-3">
+              <FormLabel>Favicon</FormLabel>
+              <FormControl>
+                <>
+                  {!field.value && (
+                    <Input
+                      type="file"
+                      accept="image/x-icon,image/png"
+                      ref={inputFileRef}
+                      onChange={(e) => handleChange(e.target.files)}
+                    />
+                  )}
+                  {field.value && (
+                    <div className="flex items-center">
+                      <div className="h-5 w-5">
+                        <img src={field.value} />
+                      </div>
```

Comment:
Can we have the `Image` component from Next here? We might want to extend `next.config.js` with the [vercel domain](https://nextjs.org/docs/pages/api-reference/components/image#domains) as well. 

I think a little border would be nice as well

```suggestion
                      <div className="h-5 w-5 border border-border rounded-sm">
                        <img src={field.value} />
                      </div>
```